source/compatibility.tex

```diff
@@ -97,7 +97,7 @@
 \end{codeblock}
 
 \howwide
-Programs that have a legitimate reason to treat string literals
+Programs that have a legitimate reason to treat \grammarterm{string-literal}s
```

Comment:
I think this means "string literal objects" not *string-literal*s.

Reply:
"string literal objects" it is.